### ISSUE TITLE
Does veraPDF check for Annex L relationships?

### ISSUE THREAD

**Issue by petervwyatt** [veraPDF/veraPDF-arlington-tools#8]
The Arlington PDF Model currently does not include the Annex L parent/child relationships. Is this something that veraPDF checks when processing a PDF 2.0 file?

**Comment by MaximPlusov** [comment]
veraPDF Arlington checker does not check Annex L relationships, but veraPDF has separate profile for ISO32005 / ISO32000, Annex L: https://github.com/veraPDF/veraPDF-validation-profiles/blob/integration/PDF_UA/ISO-32005-Tagged.xml

**Comment by petervwyatt** [comment]
AFAICT the XML of ISO TS 32005 doesn't clearly distinguish between PDF 2.0 and PDF 1.7 tag rules.

ISO 32000-2:2020 Annex L is _normative_ for all PDF 2.0 files (even without ISO TS 32005), whereas ISO 32000-1:2008 was silent on child/parent nesting rules (except for some occasional words) and thus why ISO TS 32005 was needed. I'm led to believe that for the PDF 2.0 tag set, ISO TS 32005 is identical to normative Annex L, so if the current XML is split as 2 files (one for ISO TS 32005 PDF 1.7 tags and ISO 32000-2:2020 Annex L) then we'd get both Annex L validation and ISO TS 32005 validation.

**Comment by petervwyatt** [comment]
BTW how did you generate that veraPDF XML? Did you do it via a script that processes the XLSX?

**Comment by MaximPlusov** [comment]
Yes. We use the script that processes CSV derived from the XLSX.
